pydantic/_internal/_fields.py

```diff
@@ -219,7 +222,7 @@ def collect_model_fields(  # noqa: C901
 
     if typevars_map:
         for field in fields.values():
-            field.apply_typevars_map(typevars_map, types_namespace)
+            field.apply_typevars_map(typevars_map, {})
```

Comment:
The `apply_typevars_map` signature was changed from:

`(typevars_map, types_namespace) -> None`

to

`(typevars_map, globalns = None, localns= None) -> None`.

The function does two things:
- call `eval_type_lenient` (using the ns arguments)
- call `_generics.replace_type` (using the `typevars_map` argument).

In this case, the call to `eval_type_lenient` is useless because annotations were evaluated already before in this context. In other words, we could very much replace this by a simple `_generics.replace_type` call. I was even thinking about removing this `apply_typevars_map` method, or at least give more flexibility on whether we want to evaluate annotations or not.

But anyway, I just realized I made the ns arguments optional, so I'm just going to remove the useless `{}` argument here.

Reply:
Great, thanks for the explanation here.